packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -438,9 +450,40 @@ export function getDevMiniflareOptions(
 							),
 						},
 					],
+					unsafeUseModuleFallbackService: true,
 				} satisfies WorkerOptions;
 			}),
 		],
+		unsafeModuleFallbackService(request) {
+			const url = new URL(request.url);
+			const rawSpecifier = url.searchParams.get("rawSpecifier");
+			assert(
+				rawSpecifier,
+				`Unexpected error: no specifier in request to module fallback service.`
+			);
+
+			const moduleRE = new RegExp(MODULE_PATTERN);
+			const match = moduleRE.exec(rawSpecifier);
+			assert(match, `Unexpected error: no match for module ${rawSpecifier}.`);
```

Comment:
I guess that in some future where we start using the module fallback service for other things we would convert this from an asset to some kind of multi-branch if statement.

Reply:
I think the `asserts` will still be valid and then the `moduleType` in the line below will be used to branch.